lib/traject/macros/met_csv.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+module Macros
+  # Macros for extracting values from CSV rows
+  module MetCsv
+    # @param header_or_index [String] the field header or index to accumulate
+    def column(header_or_index, options = {})
```

Comment:
Can you include `Macros::Csv` and use the `column` definition from there instead of redefining it?